api/Blob.json

```
@@ -26,6 +26,10 @@
           "ie": {
             "version_added": "10"
           },
+          "nodejs": {
+            "version_added": "15.7.0",
+            "notes": "Experimental implementation. Must be imported using require('buffer').Blob or import { Blob } from 'buffer'."
```

It's not true everywhere, but often structure this information with `"alternative_name"`.

Additionally, notes can be an array. I'd suggest two separate notes here, for consistency with similar cases:

- `"Experimental implementation."`
- `"Must be imported from the <code>buffer</code> module."`

```suggestion
            "version_added": "15.7.0",
            "alternative_name": "buffer.Blob",
            "notes": "Experimental implementation. Must be imported using require('buffer').Blob or import { Blob } from 'buffer'."
```